### PR TITLE
make Device match specification

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
cc @Emilgardis

File: svd-rs/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+- add missing fields in `Device`, require `version`, `description`, `address_unit_bits` and `width`,
+  also `scheme_version` is required, but skipped during (de)serialize
```
Comment (Emilgardis) [inline svd-rs/CHANGELOG.md:11]: Typo.

**Review by Emilgardis — CHANGES_REQUESTED** [review]

File: svd-rs/CHANGELOG.md (hunk 1 above)
Comment (burrbull) [reply svd-rs/CHANGELOG.md]: Fix if you can, please. I don't understand what you saying about.

Comment (Emilgardis) [reply svd-rs/CHANGELOG.md]: Sorry 😁

`schema_version`

Comment (burrbull) [reply svd-rs/CHANGELOG.md]: Really. Fixed and squashed

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**